core-common/src/main/java/org/apache/kylin/common/KylinConfigBase.java

```diff
@@ -2219,4 +2219,8 @@ public int getStaleCubeThresholdInDays() {
     public int getStaleJobThresholdInDays() {
         return Integer.parseInt(getOptional("kylin.tool.health-check.stale-job-threshold-days", "30"));
     }
+
+    public int getDictionarySliceEvicationThreshold() {
```

Comment:
This config should be located in the domain of "DICTIONARY & SNAPSHOT"

Reply:
done